Bottom: 5b76a802a8d39680ba270d27ab42350260065e53
Top:    131551fb9b47bfc7b909d72692fcc8dfb91ea942
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-13 01:54:54 +0100

Refresh of add-support-for-arc-segments

---

diff --git a/src/polygon.c b/src/polygon.c
index bd6352a..013b6aa 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -303,6 +303,8 @@ original_poly (PolygonType * p)
           poly_InclVertex (contour->head.prev, poly_CreateNode (v));
         }
 
+      frac_circle (..);
+
       /* Is current point last in contour? If so process it. */
       if (n == p->PointN - 1 ||
           (hole < p->HoleIndexN && n == p->HoleIndex[hole] - 1))
